shared/src/main/scala/drt/shared/Api.scala

```diff
@@ -226,13 +226,22 @@ case class ApiFlightWithSplits(apiFlight: Arrival, splits: Set[Splits], lastUpda
     val apiSplitsCi = splits.find(s => s.source == SplitSources.ApiSplitsWithHistoricalEGateAndFTPercentages && s.maybeEventType == Option(EventTypes.CI))
     val apiSplitsAny = splits.find(s => s.source == SplitSources.ApiSplitsWithHistoricalEGateAndFTPercentages)
     val predictedSplits = splits.find(s => s.source == SplitSources.PredictedSplitsWithHistoricalEGateAndFTPercentages)
+    val scenarioSplits = splits.find(s => s.source == SplitSources.ScenarioSimulationSplits)
     val historicalSplits = splits.find(_.source == SplitSources.Historical)
     val terminalSplits = splits.find(_.source == SplitSources.TerminalAverage)
 
-    List(apiSplitsDc, apiSplitsCi, apiSplitsAny, predictedSplits, historicalSplits, terminalSplits).find {
+    val apiSplits: List[Option[Splits]] = if (hasValidApi)
+      List(apiSplitsDc, apiSplitsCi, apiSplitsAny)
+    else
+      List(scenarioSplits)
+
+    val splitsForConsideration: List[Option[Splits]] = apiSplits ::: List(predictedSplits, historicalSplits, terminalSplits)
+
+    val bestSplits: Option[Splits] = splitsForConsideration.find {
```

Comment:
may be dont need define bestSplits variable as it last line of function so it will return

Reply:
👍  Thanks! I added that in to debug something and forgot to remove.